src/lib/stitching/gravity/stitching.ts

```diff
@@ -131,6 +132,26 @@ export const gravityStitchingEnvironment = (
         },
       },
       ArtistSeries: {
+        image: {
+          fragment: gql`
+          ... on ArtistSeries {
+            image_url: imageURL
+            original_height: imageHeight
+            original_width: imageWidth
+          }
+          `,
+          resolve: (parent, _args, context, info) => {
+            context.imageData = parent
```

Comment:
OIC, using `context`! 😎 

Wouldn't have thought of this. Is this a solution we've used elsewhere (other than for more "global" things like sessions, user-agent, loaders etc)?